### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by nikhilcms** [spyder-ide/spyder#9934]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/lib/python2.7/dist-packages/spyder/widgets/editor.py", line 1600, in save
    return self.save_as(index=index)
  File "/usr/lib/python2.7/dist-packages/spyder/widgets/editor.py", line 1753, in save_as
    self.set_stack_index(new_index)
  File "/usr/lib/python2.7/dist-packages/spyder/widgets/editor.py", line 1170, in set_stack_index
    self.tabs.setCurrentIndex(index)
TypeError: setCurrentIndex(self, int): argument 1 has unexpected type 'NoneType'

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 2.7.15+
* Qt Versions: 5.9.5, PyQt5 5.10.1 on Linux

## Dependencies
```
pyflakes >=0.5.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.4.2 (OK)
pandas >=0.13.1   :  0.24.2 (OK)
numpy >=1.7       :  1.16.4 (OK)
sphinx >=0.6.6    :  1.8.5 (OK)
rope >=0.9.4      :  0.10.5 (OK)
jedi >=0.9.0      :  0.11.1 (OK)
psutil >=0.3      :  5.4.2 (OK)
nbconvert >=4.0   :  5.5.0 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.5.1 (OK)
IPython >=4.0;<6.0:  5.8.0 (OK)
pylint >=0.25     :  1.8.3 (OK)

```

**Comment by ccordoba12** [comment]
This could be fixed in our latest version (3.3.6). Please update and try again.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
